### ISSUE TITLE
KeyError: 'XSRF-TOKEN'

### ISSUE THREAD

**Issue by YuruiHong** [euxcet/thulearn2018#4]
Today, when I tried to submit my homework via `learn submit <file>`, the error below occurred. It seemed that the program failed the authentication due to a missing XSRF-TOKEN:

```python
Traceback (most recent call last):
  File "/home/hyr/.local/bin/learn", line 8, in <module>
    sys.exit(main())
  File "/usr/lib/python3/dist-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/hyr/.local/lib/python3.9/site-packages/thulearn2018/learn.py", line 40, in submit
    learn.init()
  File "/home/hyr/.local/lib/python3.9/site-packages/thulearn2018/browser.py", line 29, in init
    self.set_semester()
  File "/home/hyr/.local/lib/python3.9/site-packages/thulearn2018/browser.py", line 70, in set_semester
    content = self.jh.loads(self.get(settings.semester_url))
  File "/home/hyr/.local/lib/python3.9/site-packages/thulearn2018/browser.py", line 59, in get
    '_csrf': self.session.cookies.get_dict()['XSRF-TOKEN']
KeyError: 'XSRF-TOKEN'
```

`learn reset` did not seemed to help.

**Comment by thuliu-yt16** [comment]
Sorry for the late reply! 

The 'XSRF-TOKEN' was introduced to fix the homework submit issue #3. At that time, the token was required for authentication. Maybe it is deprecated or renamed by web learning now. During the development of this package, THU web learning changed its API a few times. But now, both the two developers do not have any lessons so it is hard for us to test and update the corresponding interface. 
 
You could simply avoid using `learn submit`. Or you could create a new pull request yourself to update the current interface, which we would definitely appreciate.

**Comment by YuruiHong** [comment]
`learn submit` is still working right now, and I cannot manage to reproduce the error.🤣 I'll just close this issue.
